templates/web/index.html

```diff
@@ -29,7 +31,8 @@
     <title>SPIS</title>
   </head>
   <body>
-    <div hx-get="/hx/gallery"
+    <!-- <div hx-get="/hx/gallery" -->
+    <div hx-get="/hx/map"
```

Comment:
Instead of doing this, my idea is to keep the filter bar in the regular view, and just replace the `<div class="media-gallery">` with the map. That way we can still use the filters while using the map ...